core/build.gradle

```diff
@@ -14,7 +14,10 @@ sourceSets {
     }
     test {
         java {
-            srcDirs = ['src/test/java', 'test-gen']
+            srcDirs = ['src/test/java']
```

Comment:
I changed this because both `testFixtures` and `test` used `test-gen` as a same source directory. Intellij does not support this. Since `test` depends on `testFixtures` there is no problem.

Reply:
Oh, that explains why Intellij showed all these errors. Thanks!

Reply:
Oh, it works because the PR adds a custom implementation of `LfParsingTestHelper`, which is the only class generated in `test-gen`.

Reply:
In fact, we can disable it. See https://github.com/lf-lang/lingua-franca/pull/1858